common/azure_auth.go

```diff
@@ -17,9 +17,18 @@ import (
 )
 
 // List of management information
-const armDatabricksResourceID string = "2ff814a6-3304-4ab8-85cb-cd0e6f879c1d"
+const azureDatabricksProdLoginAppID string = "2ff814a6-3304-4ab8-85cb-cd0e6f879c1d"
+
+func (aa *DatabricksClient) GetAzureDatabricksLoginAppId() string {
+	res := azureDatabricksProdLoginAppID
```

Comment:
1) make this method "unexported" - make it start with lowecase letter (`getAzure...`)
2) `strings.Contains(aa.Host, ".staging.")` is unnecessary and it leaks internal infra references. simplify it to

```
if aa.AzureDatabricksLoginAppId != "" {
			return aa.AzureDatabricksLoginAppId
		}
return azureDatabricksProdLoginAppID
```

Reply:
1 - This method is being used in `common/azure_cli_auth.go`.

2 - Good point, Fixed it.